config/devices.yaml

```diff
@@ -0,0 +1,107 @@
+# netbox device attributes that should be used as filters for devices query.
+# these translate directly into the return fields from the netbox json API.
+# If no filter is specified, by default only devices with Active (1) status are pulled.
+filters:
+  status: ["active", "staged", "offline"]
+#  manufacturer: ['juniper']
+#  custom_fields.field1: "foo"
+
+# list of tags to filter devices in Netbox. Only devices that have this tag will be 
+# synchronized to NetSpyGlass. Default is None, which causes all devices present in Netbox to be synchronized.
+whitelist:
+#  - tag1
+#  - tag2
+
+# list of tags to blacklist from netbox. Devices that have this tag will not be synchronized to NetSpyGlass.
+# Blacklist filter is applied after the whitelist (if any). Default is None, which turns this off and causes all
+# devices that pass whitelist will be synchronized
+blacklist:
+#  - tagq
+# list of devices in nsg excluding from synchronization 
+nsg_blacklist:
+  name:
+    - 192.168.1.251
+    - 192.168.1.254
+    - craig-dev3
+    - cty1-bb01
+    - cty1-bb02
+    - cty1-dc01
+    - cty1-dc02
+    - cty1-pr01
+    - cty99-bbbb
+    - device-detail-2023
+    - qa02-001
+    - qa03-001
+    - qa_test
+    - test2
+    - testzz
+    - zqa-automation-new
+  deviceId: 
+    - 52
+    - 53
+    - 61
+  address:
+    - 0.0.0.1
+
+# Mapping device name -> snmp channels. Device name can be in regex format
+default_channels:
+  sjc12a-cr01:
```

Comment:
same as above. We should not hardcode any device names, address, channel names etc in the code that we ship. It is fine to put them in the comment as an example, but not like this where they are actually used